### PR TITLE
Support fish

### PR DESCRIPTION
**Pull request by sonod**
It corresponded to the fish.

### CODE REVIEW

File: use-fish [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+function last_version
+  curl -sL -o /dev/null -w '%{url_effective}' ""$SHA_RELEASES_URL"/latest" |
+    rev |
+    cut -f1 -d'/'|
+    rev
+end
+
+function download
+  test -z "$SHA_VERSION" && set SHA_VERSION (last_version)
```
Comment (dojineko) [inline use-fish]: `&&` and `||` are available on fish 3.0+.
it is better to use `;and` and `;or` when considering compatibility.

File: use-fish [hunk 2]
```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env fish
+# Usege:
```
Comment (dojineko) [inline use-fish]: `Usage`

Comment (sonod) [reply use-fish]: genius！

**Comment by k1LoW** [comment]
@sonod @dojineko THANK YOU !!!!!! 🐟

**Pull request merged**